### DISCUSSION TITLE
App Store [Jul 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6123] [Discord Threads]
> 🙋‍♂️ **_elvenofour_**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You should find it under global account settings   
#### https://docs.nocodb.com/setup-and-usages/account-settings#app-store  
#  
> 🙋‍♂️ **_elvenofour_**  
#### Ok, I always had the feeling that I was looking in the wrong place, but the fact that the global settings are "hidden" behind your own account surprised me. Maybe that could be optimized again, especially since I don't find the documentation very informative. Can I help you with the documentation on this point in any way?  In any case, thank you very much for your help.  
#  
> 👨‍🦲 **dstala**  
#### Any helping PRs are always welcome! 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1122593307978117191)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
